yarn.lock

```diff
@@ -966,13 +966,6 @@
     "@nicolo-ribaudo/chokidar-2" "2.1.8-no-fsevents.3"
     chokidar "^3.4.0"
 
-"@babel/code-frame@7.12.11":
```

Comment:
New Next bundles a lot of deps, so our tree in `yarn.lock` is much smaller now! 👍